pkg/cloudprovider/aws/suite_test.go

```diff
@@ -98,9 +95,11 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("Allocation", func() {
+	var ctx context.Context
```

Comment:
`ctx := context.Background()`   instead of declaring and then initializing in BeforeEach?   It's a nil context anyways so it doesn't matter, unless I'm missing something?

Reply:
I consider it best practice to not re-use a context. If we ever stop using a nil context (e.g. a test context?!), I wouldn't want to accidentally be sharing it.

Reply:
fair enough